build.xml

```diff
@@ -1976,6 +1976,7 @@
   <classpathentry kind="src" path="conf" including="hotspot_compiler"/>
   <classpathentry kind="src" output="${test.classes}" path="test/unit"/>
   <classpathentry kind="src" output="${test.classes}" path="test/long"/>
+  <classpathentry kind="src" output="${test.classes}" path="test/microbench"/>
```

Comment:
`memory` and `burn` paths are missed as well, right?

Reply:
Added along anttasks

Reply:
Good catch, it seems that `anttask` is missed for the IDEA configuration as well. 
Should we fix it here?
https://github.com/apache/cassandra/blob/trunk/ide/idea-iml-file.xml

Reply:
Apologies for the late reply I have been OOO. I don't have IDEA so I can't help much here... I have added the commit though in hope you or Benjamin can test it.